docs/tutorials/services/accessing-and-moving-robot-arm.md

```diff
@@ -19,6 +19,10 @@ cost: 8400
 no_list: true
 ---
 
+<!-- LEARNING GOALS
```

Comment:
This one doesn't sufficiently meet that goal I think - I've created a follow up ticket for this